docker/build/Dockerfile

```diff
@@ -6,7 +6,7 @@ WORKDIR /root
 RUN wget --quiet https://dl.google.com/go/go${GOLANG_VERSION}.linux-amd64.tar.gz && tar -xvf go${GOLANG_VERSION}.linux-amd64.tar.gz && mv go /usr/local
 ENV GOROOT=/usr/local/go \
     GOPATH=$HOME/go \
-    GOPROXY=https://gocenter.io
+    GOPROXY=https://gocenter.io,https://goproxy.io,direct
```

Comment:
This won't work until we upgrade to golang 1.13, will it?

Reply:
Yea, but the makefile overwrites this with `GOPROXY=direct`. We figured since we already found all the places to set this goproxy setting, and it doesn't really affect our ci process at the moment, we could leave them changed for the future. An argument could be made that that we should remove them from everywhere except for the Makefile